README.md

```diff
@@ -19,7 +19,9 @@ review process.
 2. Add the OpenAI API key as a GitHub Secret in your repository with the name `OPENAI_API_KEY`. You can find more
    information about GitHub Secrets [here](https://docs.github.com/en/actions/reference/encrypted-secrets).
 
-3. Create a `.github/workflows/main.yml` file in your repository and add the following content:
+3. Fork this repository to your account or organization
```

Comment:
Forking is not needed, this action is available in the marketplace too: https://github.com/marketplace/actions/ai-code-review-action